lib/oli_web/components/delivery/exploration_card.ex

```diff
@@ -2,21 +2,27 @@ defmodule OliWeb.Components.Delivery.ExplorationCard do
   use Phoenix.Component
 
   def render(assigns) do
+    assigns = assign(assigns, dark: Map.get(assigns, :dark, false))
+
```

Comment:
I think [component attributes](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-attributes) might be a great idea for this use case, WDTY?

```suggestion

  attr :dark, :boolean, default: false
  attr :exploration, :any

  def render(assigns) do
```